src/styles/variable_.scss

```
@@ -0,0 +1,23 @@
+$calendar_width: 726px; // Ширина календаря
+$calendar_display: flex; // Flex для расположения дней
+$cell-size: 100px; // Размер каждой ячейки
+$cell-border: 1px; // Толщина границы ячейки
+$calendar_day_width: $cell-size;
+$calendar_day_height: $cell-size;
+$calendar_border: 1px solid black;
+$calendar_day_color: #eee;
+$gray-color: #eee;
+$black-color: black;
+$font-size: 30px; // Размер шрифта для цифры
+$spacer-count: 5;
+$border-size: 1px;
+$transition-duration: 500ms;
+$day-map: (
+  sun: 0,
+  mon: 1,
+  tue: 2,
+  wed: 3,
+  thu: 4,
+  fri: 5,
+  sat: 6,
+);
```

```suggestion
$day-map: (
  mon: 0,
  tue: 1,
  wed: 2,
  thu: 3,
  fri: 4,
  sat: 5,
  sun: 6,
);
```
week should start from monday